nestjs/src/courses/courses.module.ts

```diff
@@ -59,6 +65,10 @@ import { CourseStatsController, CourseStatsService } from './stats';
     InterviewsService,
     TasksService,
     CourseStatsService,
+    {
+      provide: APP_INTERCEPTOR,
+      useClass: CacheInterceptor,
```

Comment:
My bad, tie the `CacheInterceptor` to `ScoreController` instead of `getScore` endpoint. `APP_INTERCEPTOR` fixed problem cause make all request cashable on Module level. 
Don't think that we need to have `APP_INTERCEPTOR` on app module so reverted this changes. 
Thanks